addon/data/panel.css

```diff
@@ -57,28 +58,80 @@ input[type=text] {
 */
 
 .container {
-  display: flex;
-  flex-direction: column;
-  height: 100%;
-  padding: 5px;
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  bottom: 0px;
+  right: 0px;
+  overflow: hidden;
 }
 
-img.snippet {
-  width: 100%;
+.modes-row {
+  position: absolute;
+  top: 0.5em;
+  left: 0.5em;
+  right: 0.5em;
+  text-align: center;
 }
 
-.row {
-  width: 100%;
-  border-bottom: 1px solid #999;
+.mode {
+  display: inline-block;
+  width: 33%;
+  border-bottom: 1px solid black;
+  height: 2em;
+  padding-top: 1em;
+  text-align: center;
 }
 
-.comment-input {
-  width: 100%;
-  border: 1px solid #666;
-  border-radius: 2px;
+.mode-selected {
+  border-bottom: 2px solid orange;
+  background-color: #ababab;
+}
+
+.snippets-row {
+  position: absolute;
+  bottom: 2.5em;
+  left: 0px;
+  right: 0px;
+  text-align: center;
+}
+
+.link-row {
+  position: absolute;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  height: 2em;
+  border-top: 1px solid black;
+}
+
+.link {
+  position: absolute;
+  bottom: 0.275em;
+  left: 0.5em;
+  right: 5.5em;
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.copy {
+  position: absolute;
+  bottom: 0.5em;
+  right: 0.5em;
+  width: 6em;
+}
+
+img.snippet {
+  position: absolute;
+  top: 4.5em;
+  left: 5%;
+  width: 90%;
 }
 
 .text {
+  display: none;
```

Comment:
Or just include a comment indicating why they are hidden.  I wasn't sure if they would be toggled on somehow elsewhere.